index.md

```diff
@@ -89,19 +89,55 @@ as to mitigate subjectivity in role assignment.
 
 ### [Infractions](#infractions)
 It is incumbent on members to bring issues to the attention of the community.
-An issue can't be addressed if no one else is aware it exists. Either bring up
-the issue directly with the other member or speak to some of the other members
-that you are close with. If they agree that it's an issue that needs
-addressing, the matter should be brought to the community as a whole for
-discussion.
-
-### [Sanctions](#sanctions)
-* Apologize when appropriate
-* Ameliorate the situation (e.g. repair or replace a damaged game; buy person a
-  lunch to mend things)
-
-Continued violations may result in ejection from the community, by process of
-consensus of active and long-standing members.
+An issue can't be addressed if no one else is aware it exists. Ideally, members
+should bring up the issue directly with the other member.  The next recourse is
+to initiate a formal conflict resolution proceeding.  Before doing so, it is
+highly recommended that a member speak with one or two other members about the
+matter.  If they agree that it's an issue that needs addressing, the matter
+should be brought to the attention of the Adepts and will move through the
```

Comment:
```suggestion
should be brought to the attention of the [Adepts](#contributing-members) and will move through the
```

I think it's a good idea to link back to the section about the Adepts, just in case someone needs a reminder about who they are and how to find them.